gradle.properties

```diff
@@ -14,15 +14,15 @@ minecraft_version_max=1.17
 # Forge Version Information
 javafml_min=33.0
 javafml_max=35.0
-forge_version_min=33.0.0
-forge_version=34.0.13
+forge_version_min=34.0.0
+forge_version=34.1.12
 forge_version_max=35.0.0
 
 # Mappings Information
 mappings_version=20200916-1.16.2
 
 # Build Dependencies
 mantle_version=1.6.38
-jei_version=7.1.+
-hwyla_version=1.10.10-B77
-probe_version=3.0.2-beta-5
+jei_version=7.5.0.43
```

Comment:
Restricting versions to a specific one is never a good idea for compatibility issues, unless you have a very good reason then this should be `7.5.+`

Reply:
No problem, will correct this!